.github/workflows/label-pullrequest.yml

```diff
@@ -2,14 +2,17 @@
 name: meta(labels)
 
 on:
-  pull_request:
+  pull_request_target:
 
 jobs:
   label-pullrequest:
+    permissions:
+      contents: read
+      pull-requests: write
     name: labels pull requests (frontend / backend)
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
I would be more comfortable if this explicitly checked out `master` (since `pull_request_target` is only actually scary if it checks out the PR branch)